src/docfx/build/xref/XrefResolver.cs

```diff
@@ -130,6 +131,16 @@ public XrefMapModel ToXrefMapModel()
             return model;
         }
 
+        private string RemoveSharingHost(string url, string hostName)
+        {
+            if (url.StartsWith(hostName, StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
            if (url.StartsWith(hostName + '/', StringComparison.OrdinalIgnoreCase))
```

Reply:
why do we need appending `/` with hostName here?

Reply:
`https://a.com/b` starts with `https://a.co`, but you shouldn't remove in this case 😁

Reply:
Thanks for the explanation, fair enough